packages/cli/src/__tests__/installer.test.ts

```diff
@@ -0,0 +1,337 @@
+      ).toEqual(
+        `const { Appsignal } = require("@appsignal/nodejs");\n\n` +
+          `const appsignal = new Appsignal({\n` +
+          `  active: true,\n` +
+          `  name: "MyApp",\n` +
+          `  pushApiKey: "00000000-0000-0000-0000-000000000000",\n` +
+          `});\n`
+      )
+    })
+  })
+
+  describe("when choosing to install with a configuration file, with a src directory that's actually a file", () => {
+    beforeAll(async () => {
+      mockedPrompt.mockResolvedValueOnce({
+        pushApiKey: "00000000-0000-0000-0000-000000000000",
+        name: "MyApp"
+      })
+      mockedPrompt.mockResolvedValueOnce({
+        method: "Using an appsignal.cjs configuration file."
+      })
+
+      fs.writeFileSync(src, "")
+      await installNode(tmpdir)
+    })
+
+    afterAll(() => {
+      fs.rmSync(path.join(tmpdir, "appsignal.cjs"))
+      fs.rmSync(src, { recursive: true, force: true })
+      jest.clearAllMocks()
+    })
+
+    it("it refers to appsignal.cjs the post-install instructions", () => {
+      expect(consoleLogSpy.mock.calls).toContainEqual([
+        "Writing appsignal.cjs configuration file."
+      ])
+      expect(consoleLogSpy.mock.calls).toContainEqual([
+        "    node --require './appsignal.cjs' index.js"
+      ])
+    })
+
+    it("it writes an appsignal.cjs configuration file", () => {
+      expect(
+        fs.readFileSync(path.join(tmpdir, "appsignal.cjs")).toString()
+      ).toEqual(
+        `const { Appsignal } = require("@appsignal/nodejs");\n\n` +
+          `const appsignal = new Appsignal({\n` +
+          `  active: true,\n` +
+          `  name: "MyApp",\n` +
+          `  pushApiKey: "00000000-0000-0000-0000-000000000000",\n` +
+          `});\n`
+      )
+    })
+  })
+
+  describe("when choosing to install with environment variables", () => {
+    beforeAll(async () => {
+      mockedPrompt.mockResolvedValueOnce({
+        pushApiKey: "00000000-0000-0000-0000-000000000000",
+        name: "MyApp"
+      })
+      mockedPrompt.mockResolvedValueOnce({
+        method: "Using system environment variables."
+      })
+
+      await installNode(tmpdir)
+    })
+
+    afterAll(() => {
+      jest.clearAllMocks()
+    })
+
+    it("it prints the post-install instructions", () => {
+      expect(consoleLogSpy.mock.calls).toEqual([
+        [],
+        ["📡 Demonstration sample data sent!"],
+        [],
+        [
+          "It may take a minute for the data to appear on https://appsignal.com/accounts"
+        ],
+        [],
+        ["Writing appsignal.cjs configuration file."],
+        [],
+        [`🎉 Great news! You've just installed AppSignal to your project!`],
+        [],
+        [
+          `Now, you can run your application like you normally would, but use the --require flag to load AppSignal's instrumentation before any other library:`
+        ],
+        [],
+        [`    node --require './appsignal.cjs' index.js`],
```

Comment:
```suggestion
        [`    node --require './appsignal.cjs' index.js`],
        [],
```

I think there should be an empty line here. Now it renders like:

```
    node --require './src/appsignal.cjs' index.js
You've chosen to use environment variables to configure AppSignal:
```